File: programme/sum-of-square/SumOfSquare.java

// Write a program to print sum of square.
import java.util.*;

class SumOfSquare {

    public static int squaresum(int n) {
        return (n * (n + 1) / 2) * (2 * n + 1) / 3;
    }

    public static void main(String[] args) {
        try (// Input character
                Scanner sc = new Scanner(System.in)) {
            System.out.print("Input: ");
            int n = sc.nextInt();

            // output ASCII value
            System.out.println(squaresum(n));
        }
    }
}

File: programme/sum-of-square/SumOfSquare .js
// Write a programme to print sum of square of n natural number
// Check it here : https://onecompiler.com/javascript/3y5c8z56m

var readline = require("readline"); // Process to read standard input

var rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Input  : ", function (N) {
  // Read N
  N = parseInt(N); // Convert the input into integer
  N = (N * (N + 1) * (2 * N + 1)) / 6; // Appy the equation
  console.log("Output :", N); // Printing the result
  rl.close();
});
